packages/ui/src/components/BrandingHeader/index.tsx

```diff
@@ -16,7 +16,7 @@ const BrandingHeader = ({ logo, headline, className }: Props) => {
 
   return (
     <div className={classNames(styles.container, className)}>
-      {logo && <img className={styles.logo} alt="app logo" src={logo} />}
+      {logo && <img className={styles.logo} alt="app logo" src={logo} crossOrigin="anonymous" />}
```

Comment:
should we add this for `console` images?

Reply:
Consider adding our CDN origin to the image src allow-list to the console app, like providing an env variable to config the allow-list.   So all the image src can be restricted.

Reply:
OWASP ref link provided using comments.

Reply:
will do some exclusive work on the cloud console app.

Reply:
didn't get this actually, let's discuss offline